### PR TITLE
fixes pagetree menu link and close tooltip

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- updates jQuery ui and adds effect
- closes tooltip on clicking on content

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-12 at 16 20 18|screen shot 2015-11-12 at 16 20 30|

### CODE REVIEW

**Comment by FinalAngel** [comment]
awesome :+1:

**Comment by FinalAngel** [comment]
@lory87 are you certain that styling is an issue, I see the following two errors that might also trigger that behaviour:

**on load**

``GET http://localhost:8000/static/admin/img/search.svg 404 (NOT FOUND)``

**when clicking on the menu switch button**

````
XHR finished loading: POST "http://localhost:8000/en/admin/cms/page/5/change-navigation/?language=en
Uncaught TypeError: this.effect is not a function
```

**Comment by lory87** [comment]
@FinalAngel checking. This errors have nothing to do with my change. This is another issue.

**Comment by FinalAngel** [comment]
@lory87 just saying that the two errors might actually cause the language icons to collapse, as some required js might not be triggered.

File: cms/static/cms/js/modules/cms.changelist.js [hunk 1]
```diff
@@ -211,6 +211,13 @@
                         }
                     });
                 });
+                $('html').on('touchend', function (e) {
+                    if (!$(event.target).hasClass('unpuplushed') || !$(event.target).hasClass('published') || !$(event.target).hasClass('dirty')) {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.changelist.js]: too long

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**